File: algorithm/src/main/java/baekjoon/p02869/Main.java
package baekjoon.p02869;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.nextInt(), sc.nextInt(), sc.nextInt()));
    }

    private static int solve(int a, int b, int v) {
        int target = v - a;
        int div = a - b;

        return (int) Math.ceil((double) target / div) + 1;
    }

}

File: algorithm/src/main/java/baekjoon/p02869/Main.kt
package baekjoon.p02869

import kotlin.math.ceil

fun main(args: Array<String>) {
    val inputs = readLine()!!.split(" ").map { it.toInt() }
    println(solve(inputs[0], inputs[1], inputs[2]))
}

private fun solve(a: Int, b: Int, v: Int): Int {
    val target = v - a
    val div = a - b

    return ceil(target.toDouble() / div).toInt().inc()
}